src/MissingPageRouter.php

```diff
@@ -31,11 +31,19 @@ public function getRedirectFor(Request $request)
     {
         $redirects = $this->redirector->getRedirectsFor($request);
 
-        collect($redirects)->each(function ($redirectUrl, $missingUrl) {
-            $this->router->get($missingUrl, function () use ($redirectUrl) {
-                $redirectUrl = $this->resolveRouterParameters($redirectUrl);
+        collect($redirects)->each(function ($redirects, $missingUrl) {
 
-                return redirect()->to($redirectUrl, Response::HTTP_MOVED_PERMANENTLY);
+            $this->router->get($missingUrl, function () use ($redirects) {
+
+                if (is_array($redirects)) {
```

Comment:
Generally I don't like `else` statements. Please move this logic to its own function called `normalizeRedirect` (or something better) that returns the url and status code to be used.